### PR TITLE
Typhoid vignette

### PR DESCRIPTION
**Pull request by kristinawlai**
Updated typhoid vignette data loading and started updating explanatory text.

### CODE REVIEW

File: vignettes/typhoid-seroincidence.Rmd [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+ readRDS()%>%
+  mutate(alpha = alpha*365.25, 
+         d = r-1) %>%
+  select(y1, alpha, d) 
+```
+
+##Load simulated data (lambda = .2) and prepare for model
+The simulated data are assuming a force of infection (FOI, *lambda*) of 0.2. We have selected hlye and IgG as our target measures. From the original dataset, we rename our variables to ______. Finally, we once again limit the dataset to only the variables needed for the analysis. 
+``` {r simdata}
+p.hlye.IgG  <- read_csv("simpophlyeigg.2.csv") %>%
```
Comment (d-morrison) [inline vignettes/typhoid-seroincidence.Rmd]: I think the checks are currently failing due to this line; fix a la https://r-pkgs.org/data.html#sec-data-system-file ?

File: vignettes/installation.Rmd [hunk 2]
```diff
@@ -127,19 +127,19 @@ by executing one of the following commands in the *R console*:
 ```{r, include=FALSE, tidy=FALSE}
 chunkTemplate <- "```{r, eval = FALSE}
 # OPTION A
-# Install Windows binary package \"seroincidence\" directly from internet ECDC website:
+# Install Windows binary package \"serocalculator\" directly from internet ECDC website:
```
Comment (d-morrison) [inline vignettes/installation.Rmd]: this option should be changed to point to github instead of ECDC

File: README.html
Comment (d-morrison) [inline README.html:1]: I don't think we want to track the html version of this file in git; delete before next commit?

File: README.md [hunk 3]
```diff
@@ -1,29 +1,47 @@
...
+[GitHub](https://github.com/) with:
+
+``` r
+install.packages("devtools")
+devtools::install_github("ucd-serg/serocalculator")
+```
+
+**A Note for Windows Users**
+
+Windows users will need to install Rtools, which contains a collection of tools for building and employing R packages that are still in development.
```
Comment (d-morrison) [inline README.md]: Did the version of Rtools installed by devtools not work? I can't remember what happened with that.

**Review by d-morrison — CHANGES_REQUESTED** [review]
Getting close to ready for merge! I left some in-line comments for details to iron out.

**Comment by d-morrison** [comment]
@kristinawlai I pushed some commits to the `typhoid_vignette` branch on your fork to try to fix the check errors we were getting, but there are some file conflicts that I can't resolve (since we're still operating on your fork for now). Would you please take a look and try to sort them out? There should be a "Resolve conflicts" button somewhere on the this page, which will take you to an online file editor. The html file just needs to be deleted, so that will hopefully be simple to resolve, but in the `typhoid-seroincidence.Rmd` file, there might be some strange-looking markings showing the line-by-line conflicts. Take a look at those and see if you can edit the file to how you think it should be? If you get stuck, please let me know!

**Comment by kristinawlai** [comment]
@d-morrison The changes are too complex to resolve in the web editor, so I can't see what to edit. Can you show me how to do this via command line or other best way to do this? We can discuss on our call.

**Pull request closed without merging**